benchmark/huffpost_kaggle/analytics.ipynb

```diff
@@ -1,110 +0,0 @@
-{
```

Comment:
Moved to separate repo for offline exploration

Reply:
Would be cool if you could share this repo with me

Reply:
I tried to upload it to GitHub. There are many big evaluation artifact files (> 3 GB total). I tried to use LFS, but GitHub limits me to 1GB. Should I just put the code into VC and share the artifacts over nfs at the end?